docs/security/docs/08-02-manage-static-users.md

```diff
@@ -40,7 +40,12 @@ If you want to add a new static user after Dex is installed, restart the Dex Pod
 ## Bind a user to a Role or a ClusterRole
 
 A newly created static user has no access to any resources of the cluster as there is no Role or ClusterRole bound to it.  
-By default, Kyma comes with two ClusterRoles: **kyma-admin**, which allows to get full admin access to the entire cluster, and **kyma-view**, which allows to view and list all of the resources of the cluster.
+By default, Kyma comes with the following ClusterRoles:
+ 
+- **kyma-admin**: allows to get full admin access to the entire cluster
+- **kyma-edit**: allows full access to kyma managed resources
+- **kyma-developer**: **kyma-edit** + full access to base kubernetes resources
```

Comment:
```suggestion
- **kyma-developer**:  allows full access to Kyma-managed resources and basic Kubernetes resources
```

Reply:
I think you should list these Kubernetes resources here. 
`(...)and basic Kubernetes resources, such as (...)`